getting_started/first_2d_game/02.player_scene.rst

```diff
@@ -68,8 +68,8 @@ Click on the ``Player`` node and add (:kbd:`Ctrl + A` on Windows/Linux or
 appearance and animations for our player. Notice that there is a warning symbol
 next to the node. An ``AnimatedSprite2D`` requires a :ref:`SpriteFrames
 <class_SpriteFrames>` resource, which is a list of the animations it can
-display. To create one, find the ``Sprite Frames`` property under the ``Animation`` tab in the Inspector and click
-"[empty]" -> "New SpriteFrames":
+display. Make sure ``AnimatedSprite2D`` is selected and then find the ``Sprite Frames`` property under 
+the ``Animation`` tab in the Inspector and click "[empty]" -> "New SpriteFrames":
```

Comment:
```suggestion
display. Make sure ``AnimatedSprite2D`` is selected and then find the ``Sprite Frames`` property under 
the ``Animation`` section in the Inspector and click "[empty]" -> "New SpriteFrames":
```
Preexisting problem, but the `Animation` section isn't a tab at all! It's a "section" of the inspector.

(We describe some common terminology [here](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#common-vocabulary-to-use-in-godot-s-documentation), though I think one or two of these may be out of date.)